### PR TITLE
Fix for a few JIRA issues

### PR DESCRIPTION
**Pull request by sureshsubbiah**
[TRAFODION 1462] : Change HBASE_SERIALIZATION to ON by default
    [TRAFODION 1463] : Support UPSERT into serialized table with index
    [TRAFODION 1464] : UPSERT into indexed table with idendity by default key causes inconsistency
    [TRAFODION 1465] : MERGE with select on source side into indexes table causes inconsistency
    [TRAFODION 1466] : Bulk load into seralized indexed table causes inconsistency
    [TRAFODION 1254] : Various inconsistency issues with Bulk load and upsert, changed a cqd name to finally close it.

### CODE REVIEW

File: core/sql/regress/hive/EXPECTED015 [hunk 1]
```diff
@@ -1400,10 +1397,6 @@ Task:  POPULATE INDEX  Status: Ended      ET: 00:00:12.244
 
 --- SQL operation complete.
 >>
->>cqd comp_bool_226 'on' ;
```
Comment (selvaganesang) [inline core/sql/regress/hive/EXPECTED015]: Does this mean that we are not testing with TRAF_ALLOW_RISKY_INDEX_MAINTENANCE 'ON'

**Comment by selvaganesang** [comment]
Can you please add some comments in the JIRA to explain how these issues were fixed. It would help the reviewer to associate the fix with the JIRA.

**Comment by selvaganesang** [comment]
It looks like you have updated the respective JIRA with the requested info. You may please ignore my earlier comment

**Comment by DaveBirdsall** [comment]
Changes look good to me.

File: core/sql/regress/hive/EXPECTED015 (hunk 1 above)
Comment (sureshsubbiah) [inline core/sql/regress/hive/EXPECTED015]: I will add a query to test this cqd in my next checkin.

**Comment by sureshsubbiah** [comment]
I would like to thank Hans for all help while resolving these bugs.

**Pull request merged**